lib/wrapper.pm

```diff
@@ -0,0 +1,57 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Wrapper for package installation, zypper or transactional update
+# Maintainer: QE Core <qe-core@suse.de>
+
+package wrapper;
+use Mojo::Base qw(Exporter);
+use testapi;
+use version_utils qw(is_transactional);
+use transactional qw(trup_call check_reboot_changes);
+use utils qw(zypper_call);
+
+our @EXPORT = "install_package";
+
+=head1 DESCRIPTION
+
+    Wrapper for zypper_call and trup_call
+
+=head2 install_package
+
+    install_package(packages => 'kernel-devel' [, trup_packages => 'kernel-default-base] [, zypper_packages => 'kernel-devel']);
+
+C<packages> defines packages to install for both zypper and trup call,
+different packages can be installaed on transactional system via C<trup_packages>
+or on standard system via C<zypper_packages>
+After trup_call check_reboot_changes can be executed via C<trup_check_reboot> parameter
+
+=cut
+
+sub install_package {
+    my (%args) = @_;
+    my $trup_packages = $args{trup_packages} // $args{packages};
+    my $zypper_packages = $args{zypper_packages} // $args{packages};
+    die 'At least one of packages, trup_packages, zypper_packages arguments is required' unless defined($args{packages} || $args{trup_packages} || $args{zypper_packages});
+    my $timeout = $args{timeout} // 500;
+
+    if (is_transactional) {
+        if ($args{trup_info}) {
+            record_info('install_package', $args{trup_info});
```

Comment:
If we really need to add an info box every time a package is not available, the keyword argument for it should be called `skip_trup` and `skip_zypper`. Also, the logic will be cleaner if you just return after adding the info box.

Reply:
You don't need to, but you can.